tools/docker/runner/README.md

```diff
@@ -0,0 +1,67 @@
+# Docker runner image
+
+This folder includes a Dockerfile and helper scripts which can be used to run prplMesh inside containers.
+It relies on an already built prplMesh by mapping the `build/install` directory to the container.
+
+It includes 2 helper scripts - one to generate the image `image-build.sh` and one to actually run the container - `run.sh`.
+
+## Usage
+
+Using this Dockerfile, you can run 2 containers, one with a controller+agent, the
+other with an agent only.
+The 2 containers are connected using a docker network (bridge), so can
+communicate + sniffed by running wireshark / tcpdump on the bridge from the host to see 1905
+packets.
+
+### Prerequisites
+
+build prplMesh which will be used by the containers (set cmake flag `SHARED_SERVER=OFF`)
+(build/install directory will be the only directory mapped to the
+containers). 
+
+```bash
+./maptools.py build map -f MSGLIB=zmq SHARED_SERVER=OFF BUILD_TESTS=ON CMAKE_BUILD_TYPE=Debug
+```
+
+Next, build the docker image using `./image-build.sh`.
+
+Then, create a docker network (bridge) `sudo docker network create
```

Comment:
Can't this be part of the image-build script?

I believe docker networks are persistent across reboots.

Reply:
need to check - but yes, good idea, i'll try that

Reply:
I added it to the run.sh script instead, since we support different networks now. The script first checks if the network exists and creates it if it doesn't.